daemon/networkdriver/bridge/driver.go

```diff
@@ -113,6 +114,13 @@ func InitDriver(config *Config) error {
 		addrsv6    []net.Addr
 		bridgeIPv6 = "fe80::1/64"
 	)
+
+	// try to modprobe bridge first
+	// see gh#12177
+	if out, err := exec.Command("modprobe", "bridge", "nf_nat").Output(); err != nil {
+		logrus.Errorf("Running modprobe bridge nf_nat failed with message: %s, error: %v", out, err)
```

Comment:
Will swap tomorrow makes sense

On Tuesday, April 14, 2015, Arnaud Porterie <notifications@github.com>
wrote:

> In daemon/networkdriver/bridge/driver.go
> <https://github.com/docker/docker/pull/12382#discussion_r28392972>:
>
> > @@ -113,6 +114,13 @@ func InitDriver(config *Config) error {
> >  		addrsv6    []net.Addr
> >  		bridgeIPv6 = "fe80::1/64"
> >  	)
> > +
> > +	// try to modprobe bridge first
> > +	// see gh#12177
> > +	if out, err := exec.Command("modprobe", "bridge", "nf_nat").Output(); err != nil {
> > +		logrus.Errorf("Running modprobe bridge nf_nat failed with message: %s, error: %v", out, err)
>
> I'd Warnf rather than Errorf but that's a nit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12382/files#r28392972>.
>